Assets/Scripts/DayCell.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Ksen;
+using TMPro;
+using UnityEngine.UI;
+
+public class DayCell : Cell
```

Comment:
This class is really good. Single responsibility and short scripts is good. It may need modified later based on earlier comments, but it should retain this structure when updates are made elsewhere. Good work

Reply:
Resolved